src/configuration/app-containers.md

```diff
@@ -116,11 +116,15 @@ The `type` can be:
 
 The `build` defines what will happen by default when building the application.
 
-It has a sub property `flavor` for which the possible values are:
+Its only property is `flavor`, which specifies a default set of build tasks to run. Flavors are language-specific.
 
-* `drupal` means that `drush make` will automatically run if you provide `.make` files.
-* `composer` means that `composer install` will automatically run if you provide a `composer.json` or `composer.lock` file.
-* `symfony` is an alisas for `composer`.
+* PHP - default `composer`
+  * `composer` will run `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
+  * `drupal` will run `drush make` automatically if a `.make` file is detected.
+* NodeJs
```

Comment:
You're right, we standardized that direction. I keep forgetting which one we use. Fixing.